spec/outputs/example_spec.rb

```diff
@@ -1 +1,20 @@
 require "logstash/devutils/rspec/spec_helper"
+require "logstash/outputs/example"
+require "logstash/codecs/plain"
+require "logstash/event"
+
+describe LogStash::Outputs::Example do
+  let(:sample_event) { LogStash::Event.new }
+
+  context "#receive" do
+    subject do
+      output = LogStash::Outputs::Example.new
+      output.register
+      output
+    end
+
+    it "returns hello world" do
+      expect(subject.receive(sample_event)).to eq("Event received")
```

Comment:
I mean a describe block. They are basically the same (context is an alias of describe), but usually you have many context per describe (having describe as the first one).